charters/DataPortal/charter.md

```diff
@@ -0,0 +1,79 @@
+# Data Portal
+
+
+## Description
+The Data Portal is a landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience (UX) for users to explore cellular resolution data. The Data Browser ("Explore") application provides a faceted search capability to search and download subsets of this data.
+
+## Objectives
+The objective of the Data Portal is to provide the DCP website implementation. It provides the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, and file/bundle search with public APIs at petabyte scale on public clouds.
+
+## In-scope
+### General
+* UX, visual, and branding design with input from the HCA brand working group
+* Adhere to the HCA DCP community best practices for Continuous Integration (CI), Continuous Delivery (CD), and system logging via Google Analytics, etc.
+* Production operation of the overall Data Portal website in Amazon Web Services (AWS), with exclusions listed in the out-of-scope section
+* Meet the performance scaling objectives of the DCP
```

Comment:
This reads like the Data Portal won't be responsible for scaling objectives, that they will be handed out from someone else, rather than the data portal participating in the scale testing process.

I also think this should be a given rather than need explicitly stating